src/pyird/spec/continuum.py

```diff
@@ -99,9 +109,52 @@ def normalize(df_continuum):
     Returns:
         pandas.DataFrame of 1D normalized spectrum
     """
+    ########
+    if (df_continuum['wav']>1420).any():
+        gain = 2.78 #for H band
+    else:
+        gain = 2.99 #for Y/J band
+    uncertainty_gain = 0.2
+
+    #RON = 12 #readout noise of IRD detectors: ~12e- (10min exposure)
+
+    def SNratio_uncertainty_interp(df_head,df_tail,RON):
```

Comment:
Can you add the comment? See the other functions as examples ([Google Napoleon style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)). These comments are used to make sphinx documents.